xml_handlers/build_handler.php

```diff
@@ -234,6 +240,27 @@ public function endElement($parser, $name)
                 }
             }
             if (array_key_exists($this->SubProjectName, $this->Builds)) {
+                // TODO: temporary fix for subtle, hard to track down issue
+                // BuildFailures' labels are not getting set in label2buildfailure when using new
+                // subproject xml schema, e.g. (<SubProject name="..."><Label>...</Label></SubProject>.
+                // If the error is being set because this is a SubProject then presumably we may
+                // ensure that the label exists here, and if not, create it.
+
```

Comment:
This change breaks the `multiplesubprojects` when run with Postgres. The failure output is:
```bash
1) Unexpected label associations
120: (3 labels): count: 0: rows:  at [/home/kitware/cdash/tests/test_multiplesubprojects.php line 634]
```

The test (rather, [a portion of that test](https://github.com/Kitware/CDash/pull/635/files#diff-13280adf55a5353cb42c8570f8b1a470R593)) is new to this branch. It was created to solve and test an issue identical to this one. Labels with the subproject name were also not being created and stored in the labels2test table. This change shouldn't have an affect on that table but apparently and somehow in Postgres is does.